scripts/text_normalization_dataset_files/README.txt

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
```

Comment:
The license should be CC BY 4.0, not Apache. Please add more details about the dataset: what it is, how it was created, its purpose. The dataset and its readme should be in a separate folder or this license header + info at the top of the dataset file. Otherwise, it's unclear whether this readme file is for the released dataset. Thanks!